param/__init__.py

```diff
@@ -36,12 +36,9 @@
 # only two required files.
 try:
     from .version import Version
-    _version = Version(fpath=__file__, archive_commit="$Format:%h$", reponame="param")
-    __version__ = str(_version)
+    __version__ = str(Version(fpath=__file__, archive_commit="$Format:%h$", reponame="param"))
 except:
-    import json
-    __version__ = json.load(open(os.path.join(os.path.split(__file__)[0],
-                                              '.version'), 'r'))['version_string']
+    __version__ = "0.0.0+unknown"
```

Comment:
Can't remember what we decided would be best here. `"None"`? Something else?

Reply:
``"0.0.0+unknown"`` seems good. It's PEP440 compliant, and if for whatever reason someone installs it any real version number will take precedence.

Reply:
Agreed. I think this was what we decided on as being better than ``"None"``.